src/index.ts

```diff
@@ -27,6 +27,9 @@ const safeSemver = (version: string) => {
 	// fix major.minor.patch.rev to use rev as build metadata
 	return (
 		version
+			// Replace any underscores with plusses, as they should
+			// have the same semantics
```

Comment:
How about leaving a comment that this is for the supervisor, so that we don't forget where this is used?